fix(legacy): crash when codeList not in store (#1145)
InseeFr/Pogues: CHANGELOG.md

## [Unreleased]

### Fixed

> - Questionnaire can now be loaded if a question uses a code list (or nomenclature) that does not exist anymore. It can happen if an existing question used a previous version of a nomenclature.

Reviewer: ```suggestion
- Questionnaire loading no longer crash when a question uses a code list (or nomenclature) that does not exist anymore. It could happen when an existing question used a previous version of a nomenclature.
```

Reviewer 2: typo: crashes
